src/Manager.php

```diff
@@ -32,14 +37,26 @@ class Manager
      */
     protected $_collection = 'default';
```

Comment:
There is a lot of magic string usage around `'default'`. Could we make this a class const to more easily reuse it in other places? Especially when we compare against it, typos here can easily lead to hard to find dev errors.